dagger/config/setup.go

```diff
@@ -0,0 +1,248 @@
+package config
```

Comment:
There is a bug currently when additional tags are an empty slice [] causes config loading/validation to fail :
 ```incorrect types at key images.opensearch.opensearch, type []string != []interface {}```
 Either setup should not generate these fields in the config if no additional tags are specified or the type conversion should be fixed